common/static/common/js/spec/components/paginated_view_spec.js

```diff
@@ -0,0 +1,182 @@
+define([
+    'backbone',
+    'underscore',
+    'common/js/spec_helpers/ajax_helpers',
+    'common/js/components/views/paginated_view',
+    'common/js/components/collections/paging_collection'
+], function (Backbone, _, AjaxHelpers, PaginatedView, PagingCollection) {
+    'use strict';
+    describe('PaginatedView', function () {
+        var TestCardView = Backbone.View.extend({
+            className: 'test-card',
+            tagName: 'div',
+            initialize: function () {
+                this.render();
+            },
+            render: function () {
+                this.$el.text(this.model.get('text'));
+                return this;
+            }
+        }),
```

Comment:
This indentation seems to be off, as it should be four characters further indented as this is inside a ``var``.